_layouts/event-page.html

```diff
@@ -0,0 +1,137 @@
+---
+layout: default
+---
+
+<article class="event-page">
+  <header class="event-header">
+    {% if page.featured_image %}
+    <div class="featured-image">
+      <img src="{{ page.featured_image }}" alt="{{ page.title }}">
+    </div>
+    {% endif %}
+    
+    <div class="event-meta">
+      {% if page.event_date %}
+      <time class="event-date">{{ page.event_date }}</time>
+      {% endif %}
+      
+      {% if page.location %}
+      <div class="event-location">{{ page.location }}</div>
+      {% endif %}
+      
+      {% if page.registration_status %}
+      <div class="registration-status {{ page.registration_status }}">
+        Registration: {{ page.registration_status | capitalize }}
+      </div>
+      {% endif %}
+    </div>
+  </header>
+
+  <div class="event-content">
+    {{ content }}
+  </div>
+</article>
+
+<style>
```

Comment:
We can remove all of this style and use Bootstrap classes and components.

Quite a lot to explain, but if you're ok with it, I can push some commits here.

-----

One small example: 

Instead of having a custom class for margins like this:

```css
.event-header {
     margin-bottom: 3rem;

   }
```

Bootstrap provides a bunch of utility classes, so you can write `<header class="mb-3">` instead.

Bootstrap also provides lots of button styles and everything else we'll need.

https://getbootstrap.com/docs/5.3/components/buttons/